programs/drip/src/actions/withdraw.rs

```diff
@@ -158,6 +193,11 @@ impl<'a, 'info> Executable for Withdraw<'a, 'info> {
                 execute_withdraw_b(&mut accounts.common, cpi_executor)
             }
             Withdraw::WithClosePosition { accounts } => {
+                let sol_destination = match &accounts.sol_destination {
+                    Some(sol_destination) => sol_destination.to_account_info(),
+                    None => accounts.common.withdrawer.to_account_info(),
```

Comment:
Your validation logic makes this code path unreachable

Reply:
Isn't it reachable for non admin invitations?

Reply:
For non-admin closePositions, this code path is reachable.